### DISCUSSION TITLE
Getting Ollama: 400, message='Bad Request', error only when starting a new chat and only for admin user

### DISCUSSION THREAD

**Discussion by h14turbo** [open-webui/open-webui#3742] [General]
HI, 

I am the admin user on an instance of Open-Webui. For some reason whenever I start a new chat I get the following error:

Ollama: 400, message='Bad Request', url=URL('http://host.docker.internal:11435/api/chat')

This doesn't happen if I continue an old chat. And only seem to affect the admin user account. Any ideas? Thanks in advance!

**Comment by yushijie94** [comment]
same as you..

**Comment by itknight** [comment]
可以尝试点击右上角的加号，新增一个 模型选择 deepseek

**Comment by ingaschoeyen** [comment]
same here, updated ollama, old chats dont work either
